site/helm-operator.md

```diff
@@ -41,12 +42,15 @@ helm-operator requires setup and offers customization though a multitude of flag
 
 ### Installing Helm / Tiller
 
-Generate certificates for Tiller and Flux. This will provide a CA, servercerts for tiller and client certs for helm / weave flux.
+Generate certificates for Tiller and Flux. This will provide a CA, servercerts for Tiller and client certs for Helm / Weave Flux.
+
+> **Note**: When creating the certificate for Tiller the Common Name should match the hostname you are connecting to from the Helm operator.
```

Comment:
This could perhaps be more explicit about which argument the hostname comes from, or how it's calculated. But I'm being picky.

Reply:
I had quite a hard time with finding the balance between being too explicit and not being explicit enough.

What makes it difficult is that the official Helm documentation rushes you through the generation of several certificates. Which from our perspective makes it difficult to guide them during this process.

In the end I choose for this sentence because it contains all the pieces of the puzzle for one to come to a solution when they encounter this error.

I could add something like:

> If the Common Name is misconfigured for the certificate the Helm operator will log `context deadline exceeded` errors because it is unable to validate the certificate.

to add another piece.

Reply:
Yes that's often a tricky balance. OK let's stick with it as-is. We can add some detail or troubleshooting advice in response to people having specific difficulties, should they arise.